MapboxAndroidDemo/src/main/res/layout/activity_style_fade_switch.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
```

Comment:
No need to use a relativeLayout here. Instead just use a frame layout which is less expensive.

Reply:
Fixed